docs/operators-and-hooks-ref.rst

```diff
@@ -75,6 +75,8 @@ Fundamentals
 
    * - :mod:`airflow.operators.latest_only`
      -
+   * - :mod:`airflow.providers.jdbc.operators.jdbc`
```

Comment:
The fundamentals section only includes operators that are in the Airflow core. Please look at the protocol section: https://airflow.readthedocs.io/en/latest/operators-and-hooks-ref.html#protocol-operators-and-hooks

Reply:
Thanks for pointing out this, I removed this and moved the link to the JDBC doc below